tests/Integration/tests/directives/directive-configuration-block/input/guides.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<guides xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides packages/guides-cli/resources/schema/guides.xsd">
+    <extension class="\phpDocumentor\Guides\RestructuredText\DependencyInjection\ReStructuredTextExtension">
+            <code-language-label language="php" label="CUSTOM" />
```

Comment:
I would like to see an integration test where ther4e are several labels for the same language

Reply:
I don't think that's part of the feature, the configuration block is intended to have a format per tab. so you cannot have multiple tabs with the same language. 
That's why symfony is using `php+annotations` and `php+attributes`